molecule/templates/molecule_docker.yml.j2

```diff
@@ -0,0 +1,18 @@
+---
+  ansible:
+    playbook: playbook.yml
+    user: root
+    connection: docker
+
+  docker:
+    containers:
+      - name: {{role}}-01
+        ansible_groups:
+          - group1
+        image: ubuntu
+        image_version: latest
```

Comment:
Would like to see the ability of pointing to a docker registry to get the image,